tests/Tests/400__ods_dashboard/413__ods_dashboard_user_mgmt.robot

```diff
@@ -83,6 +89,7 @@ Teardown Admin UI
     [Documentation]  Setup Default Values In User Management Settings
     Set Standard RHODS Groups Variables
     Set Default Access Groups Settings
+    # RHOSi Teardown
```

Comment:
`RHOSi Teardown` should be called in the Suite Teardown, to check availability and alerts

Reply:
yup, I forgot to re-enable it, i commented it for test reasons

Reply:
done!

Reply:
I think some tests are calling `Teardown Admin UI`  as Test Teardown, but at the end it will be called again in the Suite Teardown. I possible, try to call RHOSi Teardown only in the Suite Teardown

Reply:
right right, thank for capturing that. Fixed it!